src/app/business/recruitment/recruitment.component.html

```diff
@@ -22,15 +22,25 @@ <h1 class="infoTitle">Information Night</h1>
 
       <p class="infoText">
         September 21, 2021<br />
-        6:30 pm - 7:30 pm<br />
-        Zoom (Online)
+        5:30 pm - 6:30 pm<br />
+        Zoom (Online) <br />
+
+        <a
+          href="https://ucalgary.zoom.us/j/98241270447"
+          class="infoText"
+          style="color: blue"
+          target="_blank"
+        >
+          Zoom Link <br />
```

Comment:
Instead of it being "Zoom Link" as the link text can you put the actual link instead? In case that some users cannot access the link for whatever reason.